bot/exts/info/doc/_cog.py

```diff
@@ -460,7 +460,7 @@ async def refresh_command(self, ctx: commands.Context) -> None:
     async def clear_cache_command(
         self,
         ctx: commands.Context,
-        package_name: Union[PackageName, allowed_strings("*")]  # noqa: F722
+        package_name: Union[PackageName, Literal["*"]]  # noqa: F722
```

Comment:
The `#noqa` comments are no longer required when using `Literal`.

Reply:
^